lib/config/index.js

```diff
@@ -191,6 +191,7 @@ function checkHttpOptions (config) {
   const maxFormFileSize = bytes(cfg.maxFormFileSize);
   assert(Number.isInteger(maxFormFileSize) && maxFormFileSize >= 0, `[http] "maxFormFileSize" parameter: cannot parse "${cfg.maxFormFileSize}"`);
   cfg.maxFormFileSize = maxFormFileSize;
+  assert(typeof config.http.cookieAuthentication === 'boolean', `[http] "cookieAuthentication" parameter: invalid value "${config.http.cookieAuthentication }" (boolean expected)`);
```

Comment:
Shouldnt be handled by kerror?

Reply:
Usually for configuration error we only use assertions. `kerror` is more useful for API errors and there it's just an error thrown when Kuzzle is starting